docs/localnet.md

```diff
@@ -0,0 +1,54 @@
+# Local Testnets
+
+Prerequisites:
+```bash
+make build
+```
+
+## Single node deployment
+
+To run IPC in the local rootnet just perform the following :
+```bash
+cargo make --makefile ./infra/Makefile.toml node
+
+```
+
+It will create three docker containers (cometbft, fendermint, and eth-api).
+
+To stop run the following:
+```bash
+cargo make --makefile ./infra/Makefile.toml node-down
+```
+
+## Local 4-nodes deployment
+To run IPC in the local rootnet with 4 nodes perform the following command :
+```bash
+cargo make --makefile ./infra/Makefile.toml testnet
+
+```
+
+To stop the network:
+```bash
+cargo make --makefile ./infra/Makefile.toml testnet-down
+```
+
+The testnet contains four logical nodes. Each node consists of cometbft, fendermint, and ethapi containers.
+The testnet network is `192.167.10.0/24`.
+
+ETH-API is accessible on the following interfaces:
+- `192.167.10.10:8545`
+- `192.167.10.11:8546`
+- `192.167.10.12:8547`
+- `192.167.10.13:8548`
+
+## Development
```

Comment:
Is this the subtitle you wanted? The process below doesn't say anything about development.

Reply:
You are right.